### DISCUSSION TITLE
没想到

### DISCUSSION THREAD

**Discussion by yang7er** [rbind/yihui#1490] [General]
我是monahs BA 的学生，哈哈哈哈，我们学了很多shiny，以及blog的做法，每次写reference的时候总是能citation出来一堆YIhui Xie，哈哈哈哈没想到原来每天citation的人在这里～

**Comment by yihui** [comment]
感觉像被剿匪的人发现了我的老巢……

**Reply by yang7er** [reply]
我们都是di的学生，四舍五入算是我半个师兄了哈哈哈哈。原来刚开始学BA，上我们的独有的做Shiny和Blog的课，大家也是叫苦不迭，不知道学这个有什么用。后来突然发现有点像di和你们以及我们学校设置的这些课在不断推动shiny往前发展，让更多人知道。然后我也在小红书上被国内学生请去做过线上辅导做shiny哈哈哈哈，他们导师说这个可以直接搭建网站不用去弄前后端，用来交互展示很方便。以及，我去年修的ETC5523，有一个Assignment是用qmd做自己的blog的小作业，我当时写完感觉如果把我以前写的assignment分析弄的好看点放到我的blog上，不失为一种好的面试时，给对方看的作品集。然后就很巧，那段时间我在面试德勤的实习，说实话我的一面很糟糕，因为那会儿还没有学过机器学习，但是我们老板看了我的简历附带的blog，强烈要求要了我哈哈哈哈哈，觉得我有可视化的能力（虽然后来这部分是进去去做不太相关用Axure画demo的活），哈哈哈哈哈，说实话我的本科很一般，能进德勤一直都是我的小小梦想，可以说blog是我让我抓住这个机会得契机哈哈哈哈。 在德勤实习那会儿做了基金研究建模，画了产品原型图，都是我以前不敢想我能做到的事。

**Reply by yihui** [reply]
Di 的学生出来都是营销高手。如果我是面试官，看见求职者有自己的博客和项目网页展示的话，那也一定会为之大大加分的，毕竟在简历上把自己包装得很厉害谁都会，但弄个个人网站这么拉风的事情还不是每个人都会的，哈哈。
